.devcontainer/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ruby:2.7
+
+LABEL maintainer="Vanessasaurus <@vsoch>"
```

Comment:
Could you change it to "Amazon Web Services" to be aligned with the maintainer of the CLI? https://github.com/aws/aws-parallelcluster/blob/develop/cli/setup.py#L63